docs/usage/configuration-options.md

```diff
@@ -2244,8 +2244,15 @@ Or, to add a registry prefix to any `docker` images that do not contain an expli
 {
   "packageRules": [
     {
+      "description": "official images",
       "matchDatasources": ["docker"],
-      "matchPackagePrefix": ["^([^.]+)(\\/\\:)?$"],
+      "matchPackagePatterns": ["^[a-z-]+(:|$).*"],
```

Comment:
@setchy Are you sure they match what you expect?  `:` never occures inside `packageName` nor `depName`